scripts/watch.js

```diff
@@ -9,11 +9,11 @@
  * Watch files for changes and rebuild (copy from 'src/' to `build/`) if changed
  */
 
-const fs = require('fs');
-const { execSync } = require('child_process');
-const path = require('path');
-const chalk = require('chalk');
-const { getPackages } = require('./helpers');
+import fs from 'fs';
+import childProcess from 'child_process';
+import path from 'path';
+import chalk from 'chalk';
+import helpers from './helpers';
```

Comment:
we run scripts through regular node, without ESM loader, so please revert that as it won't work now